### ISSUE TITLE
emails with underscore don't render

### ISSUE THREAD

**Issue by cdevane** [MacDownApp/macdown#781]
does render:
    `<foobar@baz.com>`  

does not render: 
    `<foo_bar@baz.com>`

**Comment by franklinyu** [comment]
Bug in master branch of Hoedown, fixed in their develop branch.

**Comment by cdevane** [comment]
Is MacDown going to pick up this fix? Considering that all 20k+ email addresses at my university have underscores, I can't really use it until then, which is a bummer. thanks.

**Comment by franklinyu** [comment]
No, Hoedown developer seems to have abandon the `develop` branch. We are still struggling between fixing that bug ourselves (we may need to read the entire codebase) and turning to another backend. Sorry about your case.
